### ISSUE TITLE
Point Cloud and Image do not match

### ISSUE THREAD

**Issue by hchittanuru3** [waymo-research/waymo-open-dataset#223]
Hey! I had a question about the tutorial notebook. So I was using one of the `tfrecords` from the Waymo Open Dataset,  `training/segment-6813611334239274394_535_000_555_000.tfrecord` to be exact. When I try to project the point cloud on to the image in this case, I get a strange mismatch in the case of the stop sign on the right. I was wondering why this might be the case?

<img width="974" alt="Screen Shot 2020-09-17 at 5 30 45 PM" src="https://user-images.githubusercontent.com/21344054/93532243-a4ebda00-f90e-11ea-8387-55a5c391e626.png">

**Comment by peisun1115** [comment]
Did you try to use our pre-computed projection results? If that is not a problem, you can use our projection lib here. https://github.com/waymo-research/waymo-open-dataset/tree/master/third_party/camera/ops

**Comment by hchittanuru3** [comment]
No, I was following the notebook, testing with this data. How do we use this library? It's not packaged with `waymo_open_dataset`.

**Comment by peisun1115** [comment]
See my reply in https://github.com/waymo-research/waymo-open-dataset/issues/227
 I am also going to release a new version. but you can build the package on your own which should be quite reproducible with the provided docker container
